src/controllers/services.controllers.js

```diff
@@ -19,8 +20,20 @@ export const createNewCourse = async(req,res)=>{
         inscription_url:"www"
 
     }
+
+    for (const imageField of Object.keys(req.files)){
+        let file = req.files[imageField][0]
+        const res = await uploadFile(file.path)
+        data[imageField] = res.secure_url
+       
```

Comment:
question: all files will be stored in the same cloud? Also, does the path contain a single name or you will organize it by folders?

Reply:
All files will stored in cloudinary, in a folder called "pyglo-app"